### DISCUSSION TITLE
PowerShellRun Configuration

### DISCUSSION THREAD

**Discussion by jdhitsolutions** [mdgrs-mei/PowerShellRun#48] [General]
I wonder if I am missing something. As far as I can tell, any configurations I make such as adding favorite folders or enabling the history feature, are not persistent. Is the user expected to create a configuration script they run in their profile? Could there be an option to export and import a configuration?

**Comment by mdgrs-mei** [comment]
That is correct. The user is expected to write a configuration in their profile script just like PSReadLine options.

I have thought about creating an external configuration file but a ps1 file felt the best choice as PowerShell users were all familiar with it.

**Reply by jdhitsolutions** [reply]
Ok. This isn't clear in the `README` file. It might be helpful to have a complete sample.

**Reply by mdgrs-mei** [reply]
Yes, I will add this to the README. Thank you!

**Reply by mdgrs-mei** [reply]
Added in PR #57.
